lighthouse-core/lib/i18n/pre-locale/en-US.json

```diff
@@ -1,119 +1,351 @@
 {
   "lighthouse-core/audits/accessibility/accesskeys.js | description": {
-    "message": "Access keys let users quickly focus a part of the page. For proper navigation, each access key must be unique. [Learn more](https://dequeuniversity.com/rules/axe/3.1/accesskeys?application=lighthouse).",
-    "description": "Description of a Lighthouse audit that tells the user *why* they should try to pass. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation."
+    "message": "Access keys let users quickly focus a part of the page. For proper navigation, each access key must be unique. $LINK_START_0$Learn more$LINK_END_0$.",
```

Comment:
do we need to check this in? One more duplicate copy of our strings.... `en-US.json`, `pre-locale/en-US.json`, `sample_v2.json`, `sample_v2_round_trip.json`...

I guess it might be helpful to be able to see new changes in the PR diff for what we submit for translation? (though in theory the `locales/en-US.json` that *is* checked in will just not be correct if something went wrong)

Reply:
Yeah this is a super valid point, we don't need to check the pre-locales in.  They just need to be generated for upload to translators, but there's no reason they need to touch Github!